packages/cli/src/zosjobs/cancel/job/Job.handler.ts

```diff
@@ -43,9 +43,20 @@ export default class JobHandler extends ZosmfBaseHandler {
         // Get the job details
         const job: IJob = await GetJobs.getJob(this.mSession, jobid);
         // Cancel the job
-        await CancelJobs.cancelJobForJob(this.mSession, job, this.arguments.modifyVersion);
+        const response: IJobFeedback = await CancelJobs.cancelJobForJob(this.mSession, job, this.arguments.modifyVersion);
+        let message: string;
 
-        const message: string = `Successfully canceled job ${job.jobname} (${jobid})`;
+        if ((this.arguments.modifyVersion == null || this.arguments.modifyVersion === "1.0") && response.status === "0") {
+            message = `Successfully submitted request to cancel job ${job.jobname} (${jobid})`;
+        } else if (this.arguments.modifyVersion === "2.0" && response.status === "0") {
+            message = `Successfully cancelled job ${job.jobname} (${jobid})`;
```

Comment:
Is the change from `canceled` to `cancelled` intentional? Seems like the version with one L is more common in American English and the version with two L's is more common in British English 🤔

Reply:
That was not intentional. Just seems that my spell check prefers one over the other.